.gitignore

```diff
@@ -30,3 +30,7 @@ node_modules
 .DS_Store
 
 build
+
+# Locks
+yarn.lock
```

Comment:
`yarn` isn't a dependency of this module so this line can be removed.

Reply:
One of those "why not" additions to the `.gitignore`. (e.g. not everyone uses Mac OSX and we have `.DS_Store` above).